src/index.spec.js

```diff
@@ -34,43 +44,99 @@ describe('node-html-to-image', () => {
 
   it('should generate an jpeg image', async () => {
     await nodeHtmlToImage({
-      output: './image.jpg',
+      output: './generated/image.jpg',
       html: '<html></html>',
       type: 'jpeg',
     })
 
-    expect(fs.existsSync('./image.jpg')).toBe(true)
+    expect(fs.existsSync('./generated/image.jpg')).toBe(true)
   })
 
   it('should put html in output file', async () => {
     await nodeHtmlToImage({
-      output: './image.png',
+      output: './generated/image.png',
       html: '<html><body>Hello world!</body></html>'
     })
 
-    const text = await getTextFromImage()
+    const text = await getTextFromImage('./generated/image.png')
     expect(text.trim()).toBe('Hello world!')
   })
 
   it('should use handlebars to customize content', async () => {
     await nodeHtmlToImage({
-      output: './image.png',
+      output: './generated/image.png',
       html: '<html><body>Hello {{name}}!</body></html>',
       content: { name: 'Yvonnick' }
     })
 
-    const text = await getTextFromImage()
+    const text = await getTextFromImage('./generated/image.png')
     expect(text.trim()).toBe('Hello Yvonnick!')
   })
 })
 
-async function getTextFromImage() {
+describe('batch', () => {
+  beforeEach(() => {
+    fs.mkdirSync('./generated')
+  })
+
+  afterEach(() => {
+    rimraf.sync('./generated')
+  })
+
+  it('should create two images', async () => {
+    await nodeHtmlToImage({
+      type: 'png',
+      quality: 300,
+      html: '<html><body>Hello {{name}}!</body></html>',
+      content: [{ name: 'Yvonnick', output: './generated/image1.png' }, { name: 'World', output: './generated/image2.png' }]
+    })
+
+    const text1 = await getTextFromImage('./generated/image1.png')
+    expect(text1.trim()).toBe('Hello Yvonnick!')
+
+    const text2 = await getTextFromImage('./generated/image2.png')
+    expect(text2.trim()).toBe('Hello World!')
+  })
+
+  it('should return two buffers', async () => {
+    const result = await nodeHtmlToImage({
+      type: 'png',
+      quality: 300,
+      html: '<html><body>Hello {{name}}!</body></html>',
+      content: [{ name: 'Yvonnick' }, { name: 'World' }]
+    })
+
+    expect(result[0]).toBeInstanceOf(Buffer)
+    expect(result[1]).toBeInstanceOf(Buffer)
+  })
+
+  it.skip('should handle mass volume well', async () => {
```

Comment:
Are we skipping this because it would slow down the CI a lot? That would make sense 😬 I like that you left it there for making local testing easier, though 🙂

Reply:
Yes it lasts seven minutes on my laptop. It is a bit too long for the CI 👍